test/SafeProtocolManager.spec.ts

```diff
@@ -286,6 +286,27 @@ describe("SafeProtocolManager", async () => {
                     SENTINEL_MODULES,
                 ]);
             });
+
+            it("Should return empty list after disabling a only enabled plugin", async () => {
```

Comment:
i'd also test what happens in case we have multiple plugins to be safe, let's say 3 plugins and removing one in the middle

Reply:
https://github.com/5afe/safe-core-protocol/pull/72/commits/06eaf268f741375b1501ae076668fcc862bbdec2